.github/workflows/ci.yml

```diff
@@ -592,6 +593,7 @@ jobs:
         name: gss-${{ github.sha }}
         path: |
           maxgraph.tar.gz
+          graphscope_store_data_load.tar.gz
```

Comment:
It would be better to add another `upload-artifact` step to only load the `graphscope_store_data_load.tar.gz`, or else it will be package together with maxgraph.tar.gz which size is much larger. It will be convenient for us to be able to download this artifact only.

Reply:
done